### PR TITLE
add IBM WebsphereLiberty compatible prod #933

### LINKED ISSUE #933: IBM WebSphere Liberty 20.0.0.3 as a Compatible Software Product for Jakarta EE 8
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**
[Alasdair Nottingham - IBM - Eclipse Profile](https://accounts.eclipse.org/users/anottingham)

**A product name and version**
[IBM WebSphere Liberty 20.0.0.3](https://www.ibm.com/cloud/websphere-liberty)

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**
Both

**A ZIP file with a product logo with a transparent background in SVG format (optional)**
[websphereliberty.svg.zip](https://github.com/jakartaee/jakarta.ee/files/5467815/websphereliberty.svg.zip)

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**
[IBM Corporation](https://www.ibm.com)

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**
Web Profile: https://github.com/eclipse-ee4j/jakartaee-platform/issues/264
Full Platform: https://github.com/eclipse-ee4j/jakartaee-platform/issues/263

**The URL where Compatible Software Product is available for download**
https://www.ibm.com/support/fixcentral/swg/selectFixes?parent=ibm%7EWebSphere&product=ibm/WebSphere/WebSphere+Liberty&release=20.0.0.3&platform=All&function=fixId&fixids=wlp-javaee8-20.0.0.3&includeSupersedes=0

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #933 

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

**Comment by chrisguindon** [comment]
@nottycode @TanjaObradovic can you verify that all is well with this PR and confirm that we can go live?
https://deploy-preview-936--jakartaee.netlify.app/compatibility/

**Comment by NottyCode** [comment]
It looks good to me. I don’t know if it can go live, the certification issues are still open so I think that may mean they are still in review. @kwsutter or @TanjaObradovic  might know what the rules are better than me on when it can go live.

**Comment by kwsutter** [comment]
> It looks good to me. I don’t know if it can go live, the certification issues are still open so I think that may mean they are still in review. @kwsutter or @TanjaObradovic might know what the rules are better than me on when it can go live.

We need to wait for the Certification Requests to be Approved:
https://github.com/eclipse-ee4j/jakartaee-platform/issues/264
https://github.com/eclipse-ee4j/jakartaee-platform/issues/263

As far as the content... Why do some of the version identifiers have the "v" at the beginning and others don't?  We're not even consistent between Open Liberty and WebSphere Liberty.  Is that up to the teams submitting the Issues for this page or is that a missing convention that should be made consistent?  Personally, I'd like to remove the "v".

Speaking of Open Liberty, could we also get [this Issue](https://github.com/jakartaee/jakarta.ee/issues/855) resolved so that we can display both versions of Open Liberty on the Compatible Impls page?  Thanks!

**Comment by kwsutter** [comment]
The Compatibility Requests are now `Accepted`.  So, we're good to go with this PR (after cleaning up the "v" prefix on the version).

@chrisguindon We really need [that Open Liberty Issue](https://github.com/jakartaee/jakarta.ee/issues/855) resolved (preferably at the same time).  Thanks!

File: data/compatible_products.yml
Comment (kwsutter) [inline data/compatible_products.yml:94]: Remove the "v" -- just use 20.0.0.3

File: data/compatible_products.yml
Comment (kwsutter) [inline data/compatible_products.yml:137]: Remove the "v" -- just use 20.0.0.3

**Review by kwsutter — APPROVED** [review]
A couple of quick updates and this is ready to go.

**Comment by chrisguindon** [comment]
@kwsutter just to confirm I get this right:

1. We need to make the quick update to you are asking on this PR
2. We need to create a new PR to address https://github.com/jakartaee/jakarta.ee/issues/855
3. Once https://github.com/jakartaee/jakarta.ee/issues/855 is approved, we go live with both pr

Is that correct?

**Comment by flora8984461** [comment]
> @kwsutter just to confirm I get this right:
> 
> 1. We need to make the quick update to you are asking on this PR
> 2. We need to create a new PR to address #855
> 3. Once #855 is approved, we go live with both pr
> 
> Is that correct?

The PR for #855 is here https://github.com/jakartaee/jakarta.ee/pull/938 And the preview will be ready soon.

**Comment by kwsutter** [comment]
> @kwsutter just to confirm I get this right:
> 
>     1. We need to make the quick update to you are asking on this PR
> 
>     2. We need to create a new PR to address #855
> 
>     3. Once #855 is approved, we go live with both pr
> 
> 
> Is that correct?

Yes, and by tomorrow EOD if you can...  :-)  Thanks for your help!

**Comment by kwsutter** [comment]
> The PR for #855 is here #938 And the preview will be ready soon.

Thanks, @flora8984461.  I have provided comments on this PR #938 that need resolving before merging.  Thanks!

**Comment by flora8984461** [comment]
Hello Kevin @kwsutter, Thanks for your comments!

If I just add the new version, I wonder should I just add the link for the compatible or should I create a new element? Since the download link and other info are the same, I think just add the new version's compatible link is good (like in the red circle of the screenshot)? 

image
image

**Pull request merged**
